pkg/scheduler/plugins/loadaware/pod_assign_cache.go

```diff
@@ -61,9 +61,14 @@ func (p *podAssignCache) assign(nodeName string, pod *corev1.Pod) {
 		m = make(map[types.UID]*podAssignInfo)
 		p.podInfoItems[nodeName] = m
 	}
-	m[pod.UID] = &podAssignInfo{
-		timestamp: timeNowFn(),
-		pod:       pod,
+
+	if _, ok := m[pod.UID]; ok {
```

Comment:
Thanks for your contributions. Could you please add a UT for this case to avoid regression?

Reply:
Ok